src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.targets

```diff
@@ -86,6 +86,10 @@
 		<_UnoProjectSystemPackageReference Include="Uno.Fonts.OpenSans" ProjectSystem="true" />
 	</ItemGroup>
 
+	<ItemGroup Condition="$(UnoFeatures.Contains(';skia;')) OR $(UnoFeatures.Contains(';skiarenderer;')) OR $(IsDesktop) == 'true'">
```

Comment:
Doesn't this mean that it will also be included on native renderer targets with `Skia` Uno Feature included?

Reply:
This is the original intent. The idea is that this particular class would be available everwhere when skia is enabled, but would be failing as not supported when not running under the skia runtime.